sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-redis/src/main/java/com/azure/messaging/eventhubs/checkpointstore/redis/JedisRedisCheckpointStore.java

```diff
@@ -50,11 +56,35 @@ public Flux<PartitionOwnership> listOwnership(String fullyQualifiedNamespace, St
     }
 
     /**
+     * This method updates the checkpoint in the Jedis resource for a given partition.
+     *
      * @param checkpoint Checkpoint information for this partition
-     * @return Null
+     * @return The new eTag of the checkpoint that has been successfully added.
```

Comment:
The return actually is a Mono that either completes or errors when the update checkpoint operation is complete, it doesn't return any value.

Reply:
Thanks Connie! I changed that in the JavaDocs of the function:)